tests/tasks/setup/eks/awscli-cfn-lt-al2023.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-cfn-launch-template-al2023
+  namespace: scalability
+spec:
+  description: |
+    Create an EKS CFN stack to output a launch template for AL2023-based nodes.
+    This Task can be used to create an EKS CFN stack that outputs a launch template.
+    The launch template may be used for a managed nodegroup with or without a custom AMI.
+  params:
+    - name: cluster-name
+      description: EKS cluster you want to create CFN stack for.
+    - name: stack-name
+      description: Stack name you want to spin.
+    - name: region
+      default: "us-west-2"
+      description: The region where the cluster is in.
+    - name: kubernetes-version
+      default: "1.28"
+      description: The EKS version to install.
+    - name: ng-cfn-url
+      description: The url of the CFN YAML/JSON to create CFN stack for NG launch template
+    - name: endpoint
+      default: ""
+    - name: kubelet-config
+      default: "{}"
```

Comment:
please let's add description for this ? It might be obvious for us, may not be for other consumers.

Reply:
Sure, copied the description from the CFN template.